src/frontend/Lex.mll

```diff
@@ -44,6 +44,10 @@ let keywords =
     ("→", RIGHT_ARROW);
     ("𝕀", DIM);
     ("dim", DIM);
+    ("𝟚", DDIM);
+    ("ddim", DDIM);
+    ("d0", D0);
+    ("d1", D1);
```

Comment:
Any Ideas why this (and the changes in Grammar) doesn't seem to work? (at least I think this is the issue, I get the same error message of "Unbound variable" when i try `def zeroDInt : 𝟚 := d0` and `def zeroDInt : FOO := d0`)